docs/resource/os.md

```diff
@@ -49,6 +49,17 @@ In case of virtualized environments, this is the operating system as it is obser
 | `solaris` | SunOS, Oracle Solaris | ![Development](https://img.shields.io/badge/-development-blue) |
 | `windows` | Microsoft Windows | ![Development](https://img.shields.io/badge/-development-blue) |
 | `zos` | IBM z/OS | ![Development](https://img.shields.io/badge/-development-blue) |
+---
+
+**Sourcing Attribute Values:**
+
+SHOULD the value not be provided natively via the SDK the following sources can be used:
+
+| Attribute | Implementation | Location |
+| --- | --- | --- |
+| [`os.build_id`](/docs/registry/attributes/os.md) | Linux | `BUILD_ID` from `/etc/os-release` or `/usr/lib/os-release` |
```

Comment:
The previous note listed `contents of '/proc/sys/kernel/osrelease'` as a third fallback location.
Was this meant to be removed as part of this PR?